tests/bricks/test_recurrent.py

```diff
@@ -143,14 +143,17 @@ def sigmoid(x):
 
 class TestGatedRecurrent(unittest.TestCase):
     def setUp(self):
-        self.gated = GatedRecurrent(
-            dim=3, weights_init=Constant(2),
-            activation=Tanh(), gate_activation=Tanh())
+        self.gated = GatedRecurrent(dim=3,
+                                    activation=Tanh(),
+                                    gate_activation=Tanh(),
+                                    weights_init=Constant(2))
         self.gated.initialize()
-        self.reset_only = GatedRecurrent(
-            dim=3, weights_init=IsotropicGaussian(),
-            activation=Tanh(), gate_activation=Tanh(),
-            use_update_gate=False, seed=1)
+        self.reset_only = GatedRecurrent(dim=3,
```

Comment:
Although technically PEP8 allows that, please do not use this Groundhog style indentation, that is one line per one keyword argument. It takes too much space and we in Blocks sort of don't do it. Sorry for being picky!